Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:14
+
+WORKDIR /app
+
+COPY package*.json ./
+
+RUN npm install
+
+COPY . .
+EXPOSE 3000
+
+CMD ["npm", "start"]
```

Comment:
npm start is normally for development purpose only. It will just run the app in development mode with node running in the background, that's all. Better way to host a react app in the container is to build the app using npm run build command and serve it using any web servers like Nginx.